specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-11-02-preview/scheduledEvents.json

```diff
@@ -0,0 +1,175 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2023-11-02-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/scheduledEvents/{scheduledEventName}": {
+      "get": {
+        "tags": [
+          "ScheduledEvents"
+        ],
+        "operationId": "ScheduledEvents_Get",
+        "summary": "Access to this resource type is not currently available via GET. To consume please use System Topics or Azure Resource Graph.",
```

Comment:
If this doesnt exist, why bother defining the operation at all?

Instead just define the `definitions` down below for the model? We shouldn't be generating an SDK for this anyway so I don't think it matters if the swagger contains no paths?

Reply:
I am following the guide of Aydan Yumerefendi's email, and you are also in that email thread. Here is what he suggests:

"There are some caveats about adding swagger for a type that does not support CRUD operations. At present the swagger tooling requires you to define a GET endpoint so that the type can be referenced. We are working across ARM and ARG to enable GETs for such types without RPs doing any work. This is not yet available. In the mean time, you can simply document this GET as something like this: "Access to this resource type is not currently available via GET. To consume please use System Topics or Azure Resource Graph"."

Reply:
there are also caveats with defining an operation that we do not implement...
that operation will be in the SDKs unless you add special statements to exclude them in each language readme.
S360 will also look at this operation, and the automatic azure test tooling will issue GET request at this path and possibly raise ICMs on use because it does not exist?

Reply:
Removed GET, let's see if that works.